### PR TITLE
Finished building the reviews table schema in Sequelize

### PR DESCRIPTION
**Pull request by curiousYi**

### CODE REVIEW

**Comment by clocasto** [comment]
1. Do we want to validate the rating is within a certain bounds (1 - 5)?
2. I anticipate that we will be entering dates into the table as integer values in ms (based on what JS produces), so we need to either store dates as milliseconds or use a hook to convert to a date string.
3.

File: server/db/models/reviews.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+module.exports = db.define('review', {
+    name: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    writtenReview: {
+        type: Sequelize.STRING(500),
+        allowNull: false,
+        validate:{
+            len: [20, 500]  //to satisfy validatio requirement that the review has to have a minimum length
```
Comment (zekenie) [inline server/db/models/reviews.js]: 500 chars is pretty small, no?

File: server/db/models/reviews.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        }
+    },
+    rating: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+    },
+    dateWritten: {
```
Comment (zekenie) [inline server/db/models/reviews.js]: sequelize has `createdAt` on all models, might not need to add again

File: server/db/models/reviews.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        allowNull: false
+    },
+    writtenReview: {
+        type: Sequelize.STRING(500),
+        allowNull: false,
+        validate:{
+            len: [20, 500]  //to satisfy validatio requirement that the review has to have a minimum length
+        }
+    },
+    rating: {
```
Comment (zekenie) [inline server/db/models/reviews.js:13]: min max?

**Comment by nrod80** [comment]
I agree with rating validation, but I think the date is okay -- Sequelize stores a Date object, which shouldn't cause any problems

**Comment by zekenie** [comment]
Agreed but i just think its redundant with the sequelize created at

**Comment by nrod80** [comment]
looks good to me, merging now

**Pull request merged**